x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/bulk_actions/bulk_edit_rules.ts

```diff
@@ -57,15 +58,15 @@ export const bulkEditRules = async ({
   const result = await rulesClient.bulkEdit({
     ...(ids ? { ids } : { filter: enrichFilterWithRuleTypeMapping(filter) }),
     operations,
-    paramsModifier: async (ruleParams: RuleAlertType['params']) => {
+    paramsModifier: (async (ruleParams: RuleAlertType['params']) => {
       await validateBulkEditRule({
         mlAuthz,
         ruleType: ruleParams.type,
         edit: actions,
         immutable: ruleParams.immutable,
       });
       return ruleParamsModifier(ruleParams, paramsActions);
-    },
+    }) as RuleParamsModifier<RuleAlertType['params']>,
```

Comment:
We should try to avoid using `as` whenever possible because every time it is used it adds a point in the code where developers might introduce a type error in the future which in turn could be or cause a runtime error as well.